src/picknik_ur_site_config/config/config.yaml

```
@@ -6,11 +6,7 @@ hardware:
   ip: "0.0.0.0"
 
 objectives:
-  behavior_loader_plugins:
-    visual_servo_behaviors:
-      - "moveit_visual_servo::behaviors::VisualServoBehaviorLoader"
-    gibson:
-      - "gibson_behavior::GibsonBehaviorsLoader"
+  behavior_loader_plugins: []
```

I think this is of type dictionary, not array.

```suggestion
  behavior_loader_plugins: {}
```